griptape/schemas/base_schema.py

```diff
@@ -59,7 +59,7 @@ def _get_field_for_type(cls, field_type: type) -> fields.Field | fields.Nested:
             if ABC in field_class.__bases__:
                 return fields.Nested(PolymorphicSchema(inner_class=field_class), allow_none=optional)
             else:
-                return fields.Nested(cls.from_attrs_cls(field_type), allow_none=optional)
+                return fields.Nested(cls.from_attrs_cls(field_class), allow_none=optional)
```

Comment:
This change represents three hours of my life on Saturday 🤪

Reply:
Yeah this is a fun area of the framework :) Can you expand a bit on what the issue was and why this fixes it?